static/app/components/superuserStaffAccessForm.tsx

```diff
@@ -247,6 +248,20 @@ class SuperuserStaffAccessForm extends Component<Props, State> {
   }
 }
 
+const FormWithApi = withApi(SuperuserStaffAccessFormContent);
+
+export default function SuperuserStaffAccessForm({api, hasStaff}: Props) {
```

Comment:
This component has no way of receiving `api` so it will be undefined (`withApi` is what gives it to the other compoent)